src/utils/config.py

```diff
@@ -314,7 +310,7 @@ def validate_freezing_run_config(config: dict) -> Tuple[bool, bool, bool, bool]:
         run_updates_and_freeze,
         run_with_frozen_data,
     ]
-    if len([val for val in values if val==True]) > 1:
```

Comment:
I changed this so the precommit would work as it doesn't like the ==True. The unit test still passes.